pkg/daemon/ceph/client/command.go

```diff
@@ -53,6 +53,8 @@ const (
 	CommandProxyInitContainerName = "cmd-proxy"
 	// ProxyAppLabel is the label used to identify the proxy container
 	ProxyAppLabel = "rook-ceph-mgr"
+	// DefaultAutoscaleMode means the default autoscale mode of pg
+	DefaultAutoscaleMode = ""
```

Comment:
We can revert all the go changes now, right? If everything can be accomplished with the pool `parameters`, no need for a code change, right?

Reply:
@travisn yes, I haven't test on it. Maybe we can close the pr.